checkov/arm/runner.py

```diff
@@ -1,22 +1,28 @@
 import logging
 import os
+from pathlib import Path
 from typing import Optional, List, Dict, Tuple
 
+from tqdm import tqdm
+
 from checkov.arm.registry import arm_resource_registry, arm_parameter_registry
 from checkov.arm.parser import parse
 from checkov.common.output.record import Record
 from checkov.common.output.report import Report, CheckType
 from checkov.common.parallelizer.parallel_runner import parallel_runner
 from checkov.common.runners.base_runner import BaseRunner, filter_ignored_paths
+from checkov.common.util.tqdm_utils import ProgressBar
 from checkov.runner_filter import RunnerFilter
 from checkov.common.parsers.node import DictNode
 from checkov.arm.context_parser import ContextParser
 
 ARM_POSSIBLE_ENDINGS = [".json"]
+FRAMEWORK = os.path.basename(Path(__file__).parent)
 
 
 class Runner(BaseRunner):
     check_type = CheckType.ARM
+    pbar = ProgressBar(FRAMEWORK)
```

Comment:
@Eliran-Turgeman Can we leverage the `check_type` member instead of the new `FRAMEWORK` attribute?
And then initialize it in the `BaseRunner`'s constructor?